librz/util/sandbox.c

```diff
@@ -210,7 +210,6 @@ RZ_API int rz_sandbox_system(const char *x, int n) {
 		eprintf ("sandbox: system call disabled\n");
 		return -1;
 	}
-#if LIBC_HAVE_FORK
 #if LIBC_HAVE_SYSTEM
 	if (n) {
 #if APPLE_SDK_IPHONEOS
```

Comment:
```suggestion
#if APPLE_SDK_IPHONEOS && LIBC_HAVE_FORK
```

Reply:
I've actually changed the whole thing so that `rz_sys_system` implements the right thing depending on the availability of `system`/`fork` on the host machine. So now `rz_sandbox_system` just checks if sandboxing is enabled and then it calls `rz_sys_system`.

Reply:
Oh ok